projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -152,6 +152,44 @@ public function is_task_list_enabled( $id ) {
 		return $this->load_value_from_callback( $task_list, 'is_enabled_callback', null );
 	}
 
+	/**
+	 * Check if a task list was dismissed by the user.
+	 *
+	 * @param string $id Task List id.
+	 * @return bool|null True if dismissed, false if not.
+	 */
+	public function is_task_list_dismissed( $id ) {
+		$task_list_dismissed_status = $this->get_task_list_dismissed_status();
+
+		// Return true if the task list is on the dismissed status array and its value is true.
+		return isset( $task_list_dismissed_status[ $id ] ) && $task_list_dismissed_status[ $id ];
+	}
+
+	/**
+	 * Set wether a task list is dismissed or not for a site.
+	 *
+	 * @param string $id Task List id.
+	 * @param bool   $is_dismissed True if dismissed, false if not.
+	 */
+	public function set_task_list_dismissed( $id, $is_dismissed ) {
+		$task_list_dismissed_status        = $this->get_task_list_dismissed_status();
```

Comment:
We need to make sure we're getting a valid `$id` value, which suggests we may need a helper function for that.
```suggestion
	public function set_task_list_dismissed( $id, $is_dismissed ) {
		if ( empty( $id ) || ! $this->has_task_list( $id ) ) {
			return;
		}
 
		$task_list_dismissed_status  = $this->get_task_list_dismissed_status();
```